wildstore-meta/src/main/java/edu/sjsu/wildstore/meta/controller/ShareLinkController.java

```diff
@@ -212,15 +214,22 @@ public long getShareLinkCount(OAuth2AuthenticationToken oAuth2AuthenticationToke
     @DeleteMapping("/{shareId}")
```

Comment:
we should probably make this one ADMIN as well.

ideally we should delete this one and make sure everyone uses the delete with the list.